ldi-nifi/ldi-nifi-processors/ldes-client-processor/pom.xml

```diff
@@ -42,6 +42,10 @@
             <version>${testcontainers-postgresql.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>be.vlaanderen.informatievlaanderen.ldes.ldi</groupId>
+            <artifactId>ldi-common</artifactId>
+        </dependency>
```

Comment:
dependency is already added at lines 20-24

Reply:
was a rebase thing that i got from PJ's commit .. will remove thanks